README.md

```diff
@@ -5,16 +5,23 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
```

Comment:
Did you run that command?  on my machine it produces the following diff:

``` diff
@@ -5,6 +5,7 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Table of Contents**
 
+- [Emacs mode for editing Rust source code](#emacs-mode-for-editing-rust-source-code)
 - [Installation](#installation)
     - [Melpa](#melpa)
     - [straight](#straight)

```



It's not related to your change, just curious if markdown-toc-refresh-toc can be trusted at all.

Reply:
Yeah it's a little annoying, that you have to remove the first line every time you run markdown-toc-refresh-toc. But aside from this it seems to work fine.

Reply:
What I was getting at is that if its reliable, it could be added to the CI script.